.gitignore

```diff
@@ -9,3 +9,7 @@ pubspec.lock
 *~
 .pub/
 bazel-*
+.idea/
```

Comment:
Should *not* put this here. This should be in a users `.gitignore_global` file.

Reply:
Done.